docs/products/kafka/howto/flink-with-aiven-for-kafka.rst

```diff
@@ -0,0 +1,200 @@
+        props.put("ssl.keystore.location", "KEYSTORE_PATH/client.keystore.p12");
+        props.put("ssl.keystore.password", "KEY_TRUST_SECRET");
+        props.put("ssl.key.password", "KEY_TRUST_SECRET");
+
+        KafkaSource<String> source = KafkaSource.<String>builder()
+            .setBootstrapServers("APACHE_KAFKA_HOST:APACHE_KAFKA_PORT")
+            .setGroupId("test-flink-input-group")
+            .setTopics("test-flink-input")
+            .setProperties(props)
+            .setStartingOffsets(OffsetsInitializer.earliest())
+            .setValueOnlyDeserializer(new SimpleStringSchema())
+            .build();
+
+        KafkaSink<String> sink = KafkaSink.<String>builder()
+            .setBootstrapServers("APACHE_KAFKA_HOST:APACHE_KAFKA_PORT")
+            .setKafkaProducerConfig(props)
+            .setRecordSerializer(KafkaRecordSerializationSchema.builder()
+                .setTopic("test-flink-output")
+                .setValueSerializationSchema(new SimpleStringSchema())
+                .build()
+            )
+            .setDeliverGuarantee(DeliveryGuarantee.AT_LEAST_ONCE)
+            .build();
+
+         // ... processing continues here
+      }
+   
+4. Tie the Kafka sources and sinks together with the ``StringCapitalizer`` in a single processing pipeline.
+
+   .. code:: java
+
+      // ... processing continues here
+      env
+        .fromSource(source, WatermarkStrategy.noWatermarks(), "Kafka Source")
+        .map(new StringCapitalizer())
+        .sinkTo(sink);
+      env.execute("Flink Java capitalizer");
+
+Build the application
+''''''''''''''''''''''''''''''''''''
+
+From the main ``capitalizer`` folder, execute the following Maven command to build the application:
+
+.. code:: shell
+
+   mvn -DskipTests=true clean package
+
+The above command should create a ``jar`` file named ``target/capitalizer-0.0.1-SNAPSHOT.jar``.
+
+Run the applications
+''''''''''''''''''''
+
+If you have installed a `local cluster installation of Apache Flink 1.15.1 <https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/try-flink/local_installation/>`_, you can launch the job on your local machine.
+``$FLINK_DIR`` is the Flink installation directory.
+
+.. code:: shell
+
+   $FLINK_DIR/bin/flink run target/capitalizer-0.0.1-SNAPSHOT.jar
+
+You can see that the job is running in the Flink web UI at ``http://localhost:8081``.
+
+.. Note::
+
+   While this example demonstrates running on a `local installation of Apache Flink 1.15.1 <https://nightlies.apache.org/flink/flink-docs-release-1.15/docs/try-flink/local_installation/>`_, it can be applied to any cluster running Apache Flink.
+   
+   Aiven also provides a hosted :doc:`/docs/products/flink` with built-in data flow integration with Aiven for Apache Kafka®
+
+Check the produced data
+'''''''''''''''''''''''
+
+:doc:`/docs/products/kafka/howto/connect-with-command-line` describes how to configure the Kafka tools to send and receive data from the topics in this example.
```

Comment:
I believe we can remove this section and just state that the user should be able to check the result of the uppercase function in the target Kafka topic.